src/exec.rs

```diff
@@ -12,6 +12,7 @@ use crate::common::{
     wcs2zstring, write_loop, ScopeGuard,
 };
 use crate::env::{EnvMode, EnvStack, Environment, Statuses, READ_BYTE_LIMIT};
+#[cfg(FISH_USE_POSIX_SPAWN)]
 use crate::env_dispatch::use_posix_spawn;
```

Comment:
Do these *need* to be cfg'd out?

Reply:
Just to make the code free from warnings.